### ISSUE TITLE
Cannot parse StringFormat=\{0:N2\}

### ISSUE THREAD

**Issue by smaucy** [NicoVermeir/XamlStyler#94]
As mentioned in headline i get following error message:

```
Error in StylerPackage:

Cannot parse markup extension string:
"{Binding Value,
StringFormat=\{0:N2\}}"

```

This was never a problem until update to version 2.

Regards

Steffen

![bildschirmfoto 2016-04-22 um 08 11 05](https://cloud.githubusercontent.com/assets/18048237/14733283/3341f1c4-0862-11e6-8d92-ee11ee4b0cff.png)

**Comment by grochocki** [comment]
Even though it works, this is actually not properly formatted XAML. Try this:

`StringFormat={}{0:N2}`

**Comment by smaucy** [comment]
Maybe you are right. But this is the way Visual Studio Properties-Toolwindow generate StringFormat. So XAML-Style should parse it. As mentioned, it was never a problem until update.

**Comment by NicoVermeir** [comment]
fixed in version 2.1 (not in gallery yet)
I'll keep this issue open until we push the new version out, so you'll get notified

**Comment by smaucy** [comment]
Thx for reply. Glad to here from you.

Regards

Steffen

**Comment by luetm** [comment]
I can confirm the issue. Escaping the braces is correct XAML in my opinion too. Thanks for the work around!

**Comment by NicoVermeir** [comment]
fixed in 2.1
